R/init.R

```diff
@@ -7,33 +7,34 @@
 #' Create the server and UI function for the `shiny` app
 #'
 #' @description `r lifecycle::badge("stable")`
```

Comment:
I think we agreed to not to have `@title` and `@description`. Please do a project-level find, I see some instances of them.

Reply:
Yes, there are some instances where tags are used 
- places  where `r lifecycle::badge` needs to be displayed with a line break. This was previously discussed on `teal.data`.https://github.com/insightsengineering/teal.data/pull/248#discussion_r1453402595
- In some places, the description is long and split into two paragraphs.

Reply:
Got it 👍🏽